protocol/activity_center_persistence_test.go

```diff
@@ -228,6 +233,103 @@ func (s *ActivityCenterPersistenceTestSuite) Test_DeleteActivityCenterNotificati
 	s.Require().NoError(err)
 }
 
+func (s *ActivityCenterPersistenceTestSuite) Test_DeleteActivityCenterNotificationsForMessage_LastMessage() {
+	// docs: Create the temporary test-database that will be used to store chats, messages, and notifications
+	db, err := openTestDB()
+	s.Require().NoError(err)
```

Comment:
Here we're creating another test case for specifically deleting NewOneToOne and NewPrivateGroupChat notifications via matching the `LastMessage` field with the deleted message ID. However, if we decide that we no longer need to support checking the LastMessage field when deleting notifications, then we can remove this test case.